ui/src/testing/default-server-config.ts

```diff
@@ -12,6 +12,9 @@ const defaultServerConfig: ConfigResponse = {
   unsafeAllowSelfBypass: false,
   defaultFreeCreditsDollarLimit: 300,
   enableBillingUpgrade: true,
+  rasHost: 'https://stsstg.nih.gov/',
+  rasClientId: '903cfaeb-57d9-4ef6-5659-04377794ed65',
+  enableRasLoginGovLinking: true,
```

Comment:
I made this to be true by default to make sure this does not break any of the existing/future code.